eurocodedesign/geometry/steelsections/__init__.py

```diff
@@ -253,21 +254,21 @@ def _load_section_props(section_name: str) -> pd.Series:
         ValueError: occurs when there is not database for section_type
 
     Returns:
-        pd.Series: containing all the geometric data for the profile
+        pd.Series[str]: containing all the geometric data for the profile
     """
     if type(section_name) != str:
         raise ValueError("Provide the section name as a string e.g. 'IPE100'")
     else:
-        if _is_valid_type(section_name):
-            section_type = _get_section_type(section_name)
-            section_db = _import_section_database(section_type)
-            if _is_valid_section(section_name, section_db):
-                return section_db.loc[section_name]
-            else:
-                raise ValueError(f"Invalid section name: '{section_name}'")
-        else:
+        if not _is_valid_type(section_name):
```

Comment:
Why did you reorder this code block? No problem - just so I understand

Reply:
Due to reordering, lesser if else blocks are neccessary and thus lesser code bloat. While it is only a small part, I prefer the variants with lesser else blocks and quicker returns.